suite/tests/client-interface/mangle_suspend.dll.c

```diff
@@ -101,17 +98,17 @@ suspend_func()
         if (!dr_get_mcontext(drcontexts[0], &mc))
             CHECK(false, "dr_get_mcontext failed!");
         /* Check xip such that we only perform the check if we are in the loop
-         * body of the test, in order to avoid races. The magic values 7 and 24
+         * body of the test, in order to avoid races. The magic values 16 and 42
```

Comment:
Please include a comment in the test about updating here when changing its code.

Reply:
Done.